README.md

```diff
@@ -153,6 +153,8 @@ I would like to thank... <a href="https://www.microverse.org/?gclid=CjwKCAiArY2f
 <!-- LICENSE -->
 
 ## :memo: License <a name="license"></a>
-This project is [MIT](./LICENCE) licensed.
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
\ No newline at end of file
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [OPTIONAL] I also recommend that you delete the Note: 👇🏻 in your License section as these are instructions on how to complete the License section.

![image](https://user-images.githubusercontent.com/79058364/228737724-d520ee13-b88a-40b0-aac8-9127fb12226a.png)